docs/releases/1.8.0.md

```diff
@@ -0,0 +1,124 @@
+# Datafaker 1.8.0 (11 Feb 2023)
+
+The first release of 2023 of Datafaker has been created. After one year of Datafaker, this marks
+the release of the last version compatible with Java 1.8. After this, Datafaker 2.x will have Java 17
+as a minimum baseline. 
+
+Why 17, and not migrate from 8 to 11? Well, we think that Java 11 hardly has any benefits for Datafaker,
+while Java 17 might have some more interesting features, such as records, compact number formatting,
+and who knows, maybe some other features. Just to be clear: our 1.x branch will no longer be supported 
+or maintained.
+
+## New Contributors
+
+A special shout out to @kingthorin, who has contributed numerous improvements in Datafaker, and 
+who is now a co-maintainer of Datafaker. Rick, welcome to the team, and absolutely appreciate 
+your amazing work!
```

Comment:
😀 Thank you!

Reply:
No, thank you for your help! It's great to have you as part of the team, thanks again!